### PR TITLE
fix: parquet read options config class

### PR DESCRIPTION
**Pull request by sverbruggen**

### CODE REVIEW

File: libraries/dagster-delta-polars/dagster_delta_polars_tests/test_type_handler.py
Comment (ion-elgreco) [inline libraries/dagster-delta-polars/dagster_delta_polars_tests/test_type_handler.py:1]: This test should be actually in dagster-delta

Comment (sverbruggen) [reply libraries/dagster-delta-polars/dagster_delta_polars_tests/test_type_handler.py:1]: Moved it

**Review by gfvioli — DISMISSED** [review]
LGTM

File: libraries/dagster-delta-polars/dagster_delta_polars_tests/test_type_handler.py
Comment (ion-elgreco) [reply libraries/dagster-delta-polars/dagster_delta_polars_tests/test_type_handler.py:1]: The test should test the DeltaPyarrowIOManager not the Polars one

**Review by ion-elgreco — CHANGES_REQUESTED** [review]
Test needs to be changed before we can merge

File: libraries/dagster-delta-polars/dagster_delta_polars_tests/test_type_handler.py
Comment (sverbruggen) [reply libraries/dagster-delta-polars/dagster_delta_polars_tests/test_type_handler.py:1]: And it's changed to test the right one

**Review by ion-elgreco — DISMISSED** [review]

**Review by ion-elgreco — APPROVED** [review]

**Pull request merged**